lib/knapsack_pro/adapters/rspec_adapter.rb

```diff
@@ -62,6 +62,10 @@ def bind_time_tracker
                 current_test_path
               end
 
+            if example.metadata[:focus] && KnapsackPro::Adapters::RSpecAdapter.rspec_configuration.filter.rules[:focus]
+              raise "We detected a test file path #{current_test_path} with a test using metadata `:focus` tag. RSpec might not run some tests in Queue Mode (causing random tests skipping problem). Please remove `:focus` tag from your codebase. See more: https://knapsackpro.com/faq/question/rspec-is-not-running-some-tests"
```

Comment:
I think we are missing some _the_'s:
- using _the_ metadata :focus tag
- in _the_ Queue Mode
- Please remove _the_ :focus tag